src/main/java/org/folio/consortia/service/impl/UserTenantServiceImpl.java

```diff
@@ -256,7 +260,12 @@ private User getUser(UUID userId) {
     }
   }
 
-  private void createActiveUser(User user) {
+  private void createActiveUserWithPermission(User user) {
+    String permissionUserId = UUID.randomUUID().toString();
+    List<String> emptyStringList = new ArrayList<>();
+    log.info("Creating permission user with id {} for userId {}", permissionUserId, user.getId());
+    var permissionUser = PermissionUser.of(permissionUserId, user.getId(), emptyStringList);
+    permissionsClient.create(permissionUser);
```

Comment:
plz make sure that you included all required permissions in modulePermissions of POST /user-tenants in module descriptor